src/components/carousel/index.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+
+import { ArrowRight32, ArrowLeft32 } from '@carbon/icons-react';
+import { useSwipeable } from 'react-swipeable';
+import styled from 'styled-components';
+
+import Theme from 'theme';
+
+import Button from '../button';
+import Dot from '../dot';
```

Comment:
Dot might be in the carrousel compont folder, it's not really a standalone component